CONTRIBUTING.md

```diff
@@ -35,10 +41,23 @@ call a _regression_), please mention it and clarify which versions you tested
 
 **The main issue tracker is for bug reports and does not accept feature proposals.**
 
-Instead, head to the [Redot Discord](https://discord.gg/redot).
+Visit the [Redot Proposals Repo](https://github.com/Redot-Engine/redot-proposals)
+to suggest features or improvements to the engine.
 
 ## Contributing pull requests
 
+Please refer to these links for detailed guides on workflow:
+(credit to Godot for these fantastic workflow guides)
+-	[Code Style](https://docs.redotengine.org/en/stable/contributing/development/code_style_guidelines.html)
+-	[PR Workflow](https://docs.redotengine.org/en/latest/contributing/workflow/pr_workflow.html)
+-	[PR Review](https://docs.redotengine.org/en/latest/contributing/workflow/pr_review_guidelines.html)
+
+Additionally, we expect all PR contributors to be using [Pre-Commit](https://pre-commit.com/) to
+minimize CI errors and improve the workflow of the entire project greatly.
```

Comment:
pre-commit is already covered in the referenced docs, and it only simply speeds up PR work since a PR that doesn't pass pre-commit always fails in CI and should never be merged.

Reply:
Ah, gotcha, will change

Reply:
Fixed and squashed